src/payment/payment-methods/cybersource/service.js

```diff
@@ -173,26 +173,24 @@ export async function checkoutWithToken(basket, { cardHolderInfo, cardDetails })
           basketId,
         });
         throw new Error('This card holder did not pass the SDN check.');
+      } else if (errorData && errorData.redirectTo) {
+        global.location.href = errorData.redirectTo;
       } else {
         logError(error, {
           messagePrefix: 'Cybersource Submit Error',
           paymentMethod: 'Cybersource',
           paymentErrorType: 'Submit Error',
           basketId,
         });
-        if (errorData && errorData.redirectTo) {
-          global.location.href = errorData.redirectTo;
-        } else {
-          if (errorData && errorData.field_errors) {
-            // It's a field error
-            // This endpoint does not return field error data in a format we expect.  Fix it.
-            error.response.data = { // eslint-disable-line no-param-reassign
-              field_errors: normalizeFieldErrors(error.response.data.field_errors),
-            };
-          }
-          handleApiError(error);
-          throw error;
+        if (errorData && errorData.field_errors) {
```

Comment:
Does this mean that (some?) field errors are (and have been) logged as JS errors?

Reply:
huh, I suppose it would, looks like that doesn't actually happen in practice since we don't see those errors in new relic?

I wonder if the backend actually catches any data validation issues that the front-end validations don't catch.

I'll rework this since that shouldn't log either.